lib/navigation/preprocess_encoders.hpp

```diff
@@ -1,3 +1,5 @@
+#include "consts.hpp"
+
```

Comment:
If you're using optional return types, you'll need to add `#include <optional>` at the top of the file here